azurerm/internal/services/network/data_source_virtual_network_gateway_connection.go

```diff
@@ -109,6 +109,29 @@ func dataSourceArmVirtualNetworkGatewayConnection() *schema.Resource {
 				Computed: true,
 			},
 
+			"traffic_selector_policy": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"local_address_ranges": {
```

Comment:
would this make more sense as
```suggestion
						"local_address_cidrs": {
```

to be explicit on what format it is expecting?

Reply:
I used `local_address_ranges` to respect azure names (https://docs.microsoft.com/en-us/powershell/module/az.network/new-azipsectrafficselectorpolicy?view=azps-3.8.0&viewFallbackFrom=azps-3.6.1).
Also, the format can be better explicited in the documentation.